packages/rpc-methods/CHANGELOG.md

```diff
@@ -6,6 +6,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.29.0]
+### Added
+- **BREAKING:** `snap.request` is now strongly typed in line with the rpc methods in this package ([#1014](https://github.com/MetaMask/snaps-monorepo/pull/1014))
+
+### Changed
+- **BREAKING:** `wallet_snap_*` rpc method was deprecated in favor of `wallet_snap` and new caveat type, `snapIds`, was added ([#1182](https://github.com/MetaMask/snaps-monorepo/pull/1182))
```

Comment:
I don't think `wallet_snap_*` still works, so it's not really deprecated, is it?

```suggestion
- **BREAKING:** `Replace `wallet_snap_*` permission with `wallet_snap` ([#1182](https://github.com/MetaMask/snaps-monorepo/pull/1182))
  - The new permission uses a caveat-based implementation to specify the snaps that are allowed to be interacted with
```